pages/agent/v3/elastic_ci_aws/parameters.md.erb

```diff
@@ -1,4 +1,4 @@
-# Elastic CI Stack for AWS Parameters
+# Elastic CI Stack for AWS parameters
 
 The following tables list all the available template parameters. These can be
```

Comment:
This definitely needs a little more intro. What template from where. 

Something like if you're using the Cloudformation template (from ...) to create an Auto Scaling group, launch template, and host resource group for maintaining a pool of EC2 instances you can use these parameters to create an evironment to match your exact needs

@keithduncan does this sound about right?

Reply:
> What template from where.

These parameters are for the [aws-stack.yml](https://github.com/buildkite/elastic-ci-stack-for-aws/blob/master/templates/aws-stack.yml) template which we link to the deployed and versioned copies of on S3.

> and host resource group

This template doesn’t create a host resource group, that’s an Elastic Mac template thing 😄 

> you can use these parameters to create an evironment to match your exact needs

Maybe swap environment for something like "use these parameters to configure the EC2 instances to suit your needs"?